internal/filter/filter.go

```diff
@@ -107,6 +111,28 @@ func (f *Filter) FindModulesForNode(node client.Object) []reconcile.Request {
 	return reqs
 }
 
+func (f *Filter) FindPreflightsForModule(mod client.Object) []reconcile.Request {
```

Comment:
I am not sure I am getting this method:
1. Should we rename that method? Not sure the name fit what the method does.
2. Why are we using a `client.Object` and not a KMM `Module`?
3. Why do we need to set the logger with the module name if we get all the Prefligh objects blindly anyway?

Reply:
if you have a better name, i will use it. 
we are using the client.Object because this is the interface that is provided by the k8s for mapping functions of the Watch
i am setting it to module name, since this function is activated on module change.

Reply:
How about `EnqueueAllPreflightValidations`?

Reply:
works for me, fixed